packages/multicast_dns/lib/multicast_dns.dart

```diff
@@ -84,13 +84,19 @@ class MDnsClient {
   /// for the mDNS query. If not provided, defaults to either `224.0.0.251` or
   /// or `FF02::FB`.
   ///
+  /// On socket errors, the [onSocketError] handler is called with the error
+  /// object and possibly a stack trace. The [onSocketError] callback must be of
+  /// type `void Function(Object error)` or
+  /// `void Function(Object error, StackTrace)`.
```

Comment:
This mirrors the `Stream.listen` docs: https://api.flutter.dev/flutter/dart-async/Stream/listen.html